src/Components/Steps/Steps.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+import { observer } from "mobx-react";
+import styles from "./Steps.module.css";
+import type { MediaPresenter } from "../../MediaPresenter";
+import type { OutputPresenter } from "../../OutputPresenter";
+import { showError } from "../ErrorToast/ErrorToast";
+
+type Props = {
+  mediaPresenter: MediaPresenter;
+  outputPresenter: OutputPresenter;
+  openPlayerModal: () => void;
+  closePlayerModal: () => void;
+};
+
+export const Steps: React.FC<Props> = observer(
+  ({ mediaPresenter, outputPresenter, openPlayerModal, closePlayerModal }) => {
+    const handleMediaUpload = async (
+      event: React.ChangeEvent<HTMLInputElement>
+    ) => {
+      if (event.target.files === null) return;
+      const files: FileList = event.target.files;
+      Array.from(files).forEach((file: File) => {
+        if (!mediaPresenter.addFile(file)) {
+          showError(file.name + "is not an acceptable file format");
+        }
+      });
+    };
+
+    const [stepOneOpen, setStepOneOpen] = useState<boolean>(true);
+    const [stepTwoOpen, setStepTwoOpen] = useState<boolean>(false);
+    const AddFile = () => {
+      return <div></div>;
+    };
+    return (
+      <div className={styles.stepsContainer}>
+        <div
+          className={styles.steppingContainer}
+          onClick={() => {
+            if (mediaPresenter.mediaReady) {
+              if(!stepOneOpen) {
+                setStepOneOpen(true);
+              }
+              closePlayerModal();
+            }         
+          }}
```

Comment:
This would be cleaner moved into a `onStepClick` function above instead of inlining it here.